crates/packager/src/package/deb/mod.rs

```diff
@@ -133,6 +141,11 @@ fn generate_desktop_file(config: &Config, data_dir: &Path) -> crate::Result<()>
     }
 
     if let Some(protocols) = &config.deep_link_protocols {
+        if !protocols.is_empty() {
+            // Use "%U" even if file associations were already provided,
+            // as it can also accommodate file names in addition to URLs.
+            exec_arg = Some("%U");
```

Comment:
can't we use `%U` for both? will that make file associations be passed as `file://path/to/file`? because I know on macOS you can either handle file associations or handle urls and if you chose urls, file associations will be passed as `file://<path>` URL, hopefully this is the case on Linux too?

Reply:
> can't we use `%U` for both?
> 
> if you chose urls, file associations will be passed as `file://<path>` URL, hopefully this is the case on Linux too?

I _think_ so, yes, that is how I understood the behavior to work based on the `.desktop` spec.

My initial line of thinking was that it'd be more straightforward to pass in file path arguments if the app dev specified only file associations, as that would be closer to their expectations (based on regular CLI app behavior). For example, an image viewing app like `Preview` or `evince` likely wouldn't expect to deal with or support URLs, but rather file paths.

But anyway, yea if you think that always passing in a URL is a better default for cargo-packager, I'm not opposed to it and can make that change.

Reply:
I see, it is fine to keep then, I was confirming because deep links override file associations

Reply:
Sounds good. It's ultimately your choice; either one works for us, but I felt like the best option was using "%U" if both deep links and file associations are provided, since the spec doesn't allow for both "%U" and "%F".